frontend/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "ng test --code-coverage --no-progress --no-watch",
     "bundlesize": "bundlesize",
     "codecov": "codecov",
-    "lint": "ng lint",
+    "lint": "ng lint > tslint-result.txt",
```

Comment:
AFAIK, there is no other way to have it generate a file (see https://github.com/angular/angular-cli/issues/4959). I want a file to be generated otherwise gradle can't mark the task up-to-date. If it has been run before and nothing has changed.

Reply:
I get it. I now remember that I use tslint directly in ng2ninja to output to a file https://github.com/Ninja-Squad/ng2ninja/blob/master/src/tslint.ts#L9